cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -331,17 +335,23 @@ def is_disposable_capital_eligible(self):
         return self.disposable_capital_assets <= limit
 
     def is_eligible(self):
-        cfe_request_dict = self._translate_case()
-
         # this line fails the lint check as we're ignoring the response
```

Comment:
this comment is a bit distracting - best to use suffix the line with `# noqa: F401` (or whatever the code is for the lint check to ignore)